src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+$square-size: 100px;
+$gap-size: 1px;
+$calendar-padding: 10px;
+$cell-color: #eee;
+$cell-size: 100px;
+$calendar-width: (7 * $cell-size) + $gap-size + $calendar-padding;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  margin: 100px auto;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)